src/core/Directus/Database/Exception/CollectionNotFoundException.php

```diff
@@ -6,7 +6,7 @@
 
 class CollectionNotFoundException extends NotFoundException
 {
-    const ERROR_CODE = 200;
+    const ERROR_CODE = 404;
```

Comment:
Nice catch @danielfrentz!

Reply:
Actually @danielfrentz I am so sorry I let you waste your time.

This is the error code, not the http status code.